2.24.0alpha3 is broken: `Field 'onlyNotifyIfNoReviews' is required for type with serial name 'com.ichi2.anki.reviewreminders.ReviewReminder', but it was missing`

### Checked for duplicates?

- [x] This issue is not a duplicate

### Does it also happen in the desktop version?

- [x] This bug does not occur in the latest version of Anki Desktop

### What are the steps to reproduce this bug?

I can't open the parallel A version!

### Expected behaviour

It works.

### Debug info

```text
v2.24.0alpha3

This is from the Google Play 2.23.1 version, in case device info is helpful:

AnkiDroid Version = 2.23.1 (3f3133b496b531383e9a6494f5a7704634935308)  
Backend Version = 0.1.62-anki25.09.2 (25.09.2 3890e12c9e48c028c3f12aa58cb64bd9f8895e30)  
Android Version = 13 (SDK 33)  
ProductFlavor = play  
Device Info = Xiaomi | POCO | mondrian | mondrian_global | 23013PC75G | qcom  
Webview User Agent = Mozilla/5.0 (Linux; Android 13; 23013PC75G Build/TKQ1.220905.001; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/143.0.7499.192 Mobile Safari/537.36  
ACRA UUID = 3792582f-98ac-430e-ae42-4ace0156bf29  
FSRS = 5.1.0 (Enabled: true)  
Crash Reports Enabled = true
```

### (Optional) Anything else you want to share?

_No response_

### Research

- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

```
FATAL EXCEPTION: main
Process: com.ichi2.anki.a, PID: 24088
java.lang.RuntimeException: Unable to create application com.ichi2.anki.AnkiDroidApp: kotlinx.serialization.MissingFieldException: Field 'onlyNotifyIfNoReviews' is required for type with serial name 'com.ichi2.anki.reviewreminders.ReviewReminder', but it was missing at path: $['ReviewReminderId(value=3)']
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7087)
	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2207)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:210)
	at android.os.Looper.loop(Looper.java:299)
	at android.app.ActivityThread.main(ActivityThread.java:8247)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:559)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:954)
Caused by: kotlinx.serialization.MissingFieldException: Field 'onlyNotifyIfNoReviews' is required for type with serial name 'com.ichi2.anki.reviewreminders.ReviewReminder', but it was missing at path: $['ReviewReminderId(value=3)']
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:95)
	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.java:43)
	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableElement(AbstractDecoder.java:70)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableElement(StreamingJsonDecoder.kt:168)
	at kotlinx.serialization.encoding.CompositeDecoder.decodeSerializableElement$default(Decoding.kt:539)
	at kotlinx.serialization.internal.MapLikeSerializer.readElement(MapLikeSerializer.java:111)
	at kotlinx.serialization.internal.MapLikeSerializer.readElement(MapLikeSerializer.java:84)
	at kotlinx.serialization.internal.AbstractCollectionSerializer.readElement$default(CollectionSerializers.kt:51)
	at kotlinx.serialization.internal.AbstractCollectionSerializer.merge(CollectionSerializers.kt:36)
	at kotlinx.serialization.internal.AbstractCollectionSerializer.deserialize(CollectionSerializers.kt:43)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
	at kotlinx.serialization.json.Json.decodeFromString(Json.kt:149)
	at com.ichi2.anki.reviewreminders.ReviewRemindersDatabase.decodeJson(ReviewRemindersDatabase.kt:380)
	at com.ichi2.anki.reviewreminders.ReviewRemindersDatabase.getRemindersForKey(ReviewRemindersDatabase.kt:238)
	at com.ichi2.anki.reviewreminders.ReviewRemindersDatabase.getAllAppWideReminders(ReviewRemindersDatabase.kt:253)
	at com.ichi2.anki.services.AlarmManagerService$Companion.scheduleAllEnabledReviewReminderNotifications(AlarmManagerService.java:226)
	at com.ichi2.anki.services.AlarmManagerService$Companion.scheduleAllNotifications(AlarmManagerService.java:280)
	at com.ichi2.anki.AnkiDroidApp.onCreate(AnkiDroidApp.kt:206)
	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1278)
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7081)
	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source) 
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2207) 
	at android.os.Handler.dispatchMessage(Handler.java:106) 
	at android.os.Looper.loopOnce(Looper.java:210) 
	at android.os.Looper.loop(Looper.java:299) 
	at android.app.ActivityThread.main(ActivityThread.java:8247) 
	at java.lang.reflect.Method.invoke(Native Method) 
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:559) 
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:954) 
Caused by: kotlinx.serialization.MissingFieldException: Field 'onlyNotifyIfNoReviews' is required for type with serial name 'com.ichi2.anki.reviewreminders.ReviewReminder', but it was missing
	at kotlinx.serialization.MissingFieldException.<init>(SerializationExceptions.kt:98)
	at kotlinx.serialization.internal.PluginExceptionsKt.throwMissingFieldException(PluginExceptions.kt:20)
	at com.ichi2.anki.reviewreminders.ReviewReminder.<init>(ReviewReminder.kt:190)
	at com.ichi2.anki.reviewreminders.ReviewReminder.<init>(ReviewReminder.kt)
	at com.ichi2.anki.reviewreminders.ReviewReminder$$serializer.deserialize(ReviewReminder.kt:190)
	at com.ichi2.anki.reviewreminders.ReviewReminder$$serializer.deserialize(ReviewReminder.kt:190)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.java:43) 
	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableElement(AbstractDecoder.java:70) 
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableElement(StreamingJsonDecoder.kt:168) 
	at kotlinx.serialization.encoding.CompositeDecoder.decodeSerializableElement$default(Decoding.kt:539) 
	at kotlinx.serialization.internal.MapLikeSerializer.readElement(MapLikeSerializer.java:111) 
	at kotlinx.serialization.internal.MapLikeSerializer.readElement(MapLikeSerializer.java:84) 
	at kotlinx.serialization.internal.AbstractCollectionSerializer.readElement$default(CollectionSerializers.kt:51) 
	at kotlinx.serialization.internal.AbstractCollectionSerializer.merge(CollectionSerializers.kt:36) 
	at kotlinx.serialization.internal.AbstractCollectionSerializer.deserialize(CollectionSerializers.kt:43) 
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69) 
	at kotlinx.serialization.json.Json.decodeFromString(Json.kt:149) 
	at com.ichi2.anki.reviewreminders.ReviewRemindersDatabase.decodeJson(ReviewRemindersDatabase.kt:380) 
	at com.ichi2.anki.reviewreminders.ReviewRemindersDatabase.getRemindersForKey(ReviewRemindersDatabase.kt:238) 
	at com.ichi2.anki.reviewreminders.ReviewRemindersDatabase.getAllAppWideReminders(ReviewRemindersDatabase.kt:253) 
	at com.ichi2.anki.services.AlarmManagerService$Companion.scheduleAllEnabledReviewReminderNotifications(AlarmManagerService.java:226) 
	at com.ichi2.anki.services.AlarmManagerService$Companion.scheduleAllNotifications(AlarmManagerService.java:280) 
	at com.ichi2.anki.AnkiDroidApp.onCreate(AnkiDroidApp.kt:206) 
	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1278) 
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7081) 
	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source) 
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2207) 
	at android.os.Handler.dispatchMessage(Handler.java:106) 
	at android.os.Looper.loopOnce(Looper.java:210) 
	at android.os.Looper.loop(Looper.java:299) 
	at android.app.ActivityThread.main(ActivityThread.java:8247) 
	at java.lang.reflect.Method.invoke(Native Method) 
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:559) 
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:954) 

```

@ericli3690 FYI

* 2.24alpha1
  * Create a few reminders with various parameters
* Update to 2.24alpha3  
  * Crash on open

```
FATAL EXCEPTION: main
Process: com.ichi2.anki.debug, PID: 15546
java.lang.RuntimeException: Unable to create application com.ichi2.anki.AnkiDroidApp: kotlinx.serialization.MissingFieldException: Field 'onlyNotifyIfNoReviews' is required for type with serial name 'com.ichi2.anki.reviewreminders.ReviewReminder', but it was missing at path: $['ReviewReminderId(value=0)']
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7510)
	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2416)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loopOnce(Looper.java:232)
	at android.os.Looper.loop(Looper.java:317)
	at android.app.ActivityThread.main(ActivityThread.java:8705)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:886)
Caused by: kotlinx.serialization.MissingFieldException: Field 'onlyNotifyIfNoReviews' is required for type with serial name 'com.ichi2.anki.reviewreminders.ReviewReminder', but it was missing at path: $['ReviewReminderId(value=0)']
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:95)
	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.kt:43)
	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableElement(AbstractDecoder.kt:70)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableElement(StreamingJsonDecoder.kt:168)
	at kotlinx.serialization.encoding.CompositeDecoder.decodeSerializableElement$default(Decoding.kt:539)
	at kotlinx.serialization.internal.MapLikeSerializer.readElement(CollectionSerializers.kt:111)
	at kotlinx.serialization.internal.MapLikeSerializer.readElement(CollectionSerializers.kt:84)
	at kotlinx.serialization.internal.AbstractCollectionSerializer.readElement$default(CollectionSerializers.kt:51)
	at kotlinx.serialization.internal.AbstractCollectionSerializer.merge(CollectionSerializers.kt:36)
	at kotlinx.serialization.internal.AbstractCollectionSerializer.deserialize(CollectionSerializers.kt:43)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
	at kotlinx.serialization.json.Json.decodeFromString(Json.kt:149)
	at com.ichi2.anki.reviewreminders.ReviewRemindersDatabase.decodeJson(ReviewRemindersDatabase.kt:380)
	at com.ichi2.anki.reviewreminders.ReviewRemindersDatabase.getRemindersForKey(ReviewRemindersDatabase.kt:238)
	at com.ichi2.anki.reviewreminders.ReviewRemindersDatabase.getAllAppWideReminders(ReviewRemindersDatabase.kt:253)
	at com.ichi2.anki.services.AlarmManagerService$Companion.scheduleAllEnabledReviewReminderNotifications(AlarmManagerService.kt:226)
	at com.ichi2.anki.services.AlarmManagerService$Companion.scheduleAllNotifications(AlarmManagerService.kt:280)
	at com.ichi2.anki.AnkiDroidApp.onCreate(AnkiDroidApp.kt:206)
	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1386)
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7504)
	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0) 
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2416) 
	at android.os.Handler.dispatchMessage(Handler.java:107) 
	at android.os.Looper.loopOnce(Looper.java:232) 
	at android.os.Looper.loop(Looper.java:317) 
	at android.app.ActivityThread.main(ActivityThread.java:8705) 
	at java.lang.reflect.Method.invoke(Native Method) 
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580) 
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:886) 
Caused by: kotlinx.serialization.MissingFieldException: Field 'onlyNotifyIfNoReviews' is required for type with serial name 'com.ichi2.anki.reviewreminders.ReviewReminder', but it was missing
	at kotlinx.serialization.internal.PluginExceptionsKt.throwMissingFieldException(PluginExceptions.kt:20)
	at com.ichi2.anki.reviewreminders.ReviewReminder.<init>(ReviewReminder.kt:190)
	at com.ichi2.anki.reviewreminders.ReviewReminder.<init>(Unknown Source:6)
	at com.ichi2.anki.reviewreminders.ReviewReminder$$serializer.deserialize(ReviewReminder.kt:190)
	at com.ichi2.anki.reviewreminders.ReviewReminder$$serializer.deserialize(ReviewReminder.kt:190)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
	... 28 more
exceptionIsUnwanted - examining RuntimeException
exceptionIsUnwanted - exception was wanted
sendAnalyticsException() description/fatal: kotlinx.serialization.MissingFieldException: Field 'onlyNotifyIfNoReviews' is required for type with serial name 'com.ichi2.anki.reviewreminders.ReviewReminder', but it was missing/true
ACRA caught a RuntimeException for com.ichi2.anki.debug
java.lang.RuntimeException: Unable to create application com.ichi2.anki.AnkiDroidApp: kotlinx.serialization.MissingFieldException: Field 'onlyNotifyIfNoReviews' is required for type with serial name 'com.ichi2.anki.reviewreminders.ReviewReminder', but it was missing at path: $['ReviewReminderId(value=0)']
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7510)
	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2416)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loopOnce(Looper.java:232)
	at android.os.Looper.loop(Looper.java:317)
	at android.app.ActivityThread.main(ActivityThread.java:8705)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:886)
Caused by: kotlinx.serialization.MissingFieldException: Field 'onlyNotifyIfNoReviews' is required for type with serial name 'com.ichi2.anki.reviewreminders.ReviewReminder', but it was missing at path: $['ReviewReminderId(value=0)']
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:95)
	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.kt:43)
	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableElement(AbstractDecoder.kt:70)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableElement(StreamingJsonDecoder.kt:168)
	at kotlinx.serialization.encoding.CompositeDecoder.decodeSerializableElement$default(Decoding.kt:539)
	at kotlinx.serialization.internal.MapLikeSerializer.readElement(CollectionSerializers.kt:111)
	at kotlinx.serialization.internal.MapLikeSerializer.readElement(CollectionSerializers.kt:84)
	at kotlinx.serialization.internal.AbstractCollectionSerializer.readElement$default(CollectionSerializers.kt:51)
	at kotlinx.serialization.internal.AbstractCollectionSerializer.merge(CollectionSerializers.kt:36)
	at kotlinx.serialization.internal.AbstractCollectionSerializer.deserialize(CollectionSerializers.kt:43)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
	at kotlinx.serialization.json.Json.decodeFromString(Json.kt:149)
	at com.ichi2.anki.reviewreminders.ReviewRemindersDatabase.decodeJson(ReviewRemindersDatabase.kt:380)
	at com.ichi2.anki.reviewreminders.ReviewRemindersDatabase.getRemindersForKey(ReviewRemindersDatabase.kt:238)
	at com.ichi2.anki.reviewreminders.ReviewRemindersDatabase.getAllAppWideReminders(ReviewRemindersDatabase.kt:253)
	at com.ichi2.anki.services.AlarmManagerService$Companion.scheduleAllEnabledReviewReminderNotifications(AlarmManagerService.kt:226)
	at com.ichi2.anki.services.AlarmManagerService$Companion.scheduleAllNotifications(AlarmManagerService.kt:280)
	at com.ichi2.anki.AnkiDroidApp.onCreate(AnkiDroidApp.kt:206)
	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1386)
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7504)
	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0) 
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2416) 
	at android.os.Handler.dispatchMessage(Handler.java:107) 
	at android.os.Looper.loopOnce(Looper.java:232) 
	at android.os.Looper.loop(Looper.java:317) 
	at android.app.ActivityThread.main(ActivityThread.java:8705) 
	at java.lang.reflect.Method.invoke(Native Method) 
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580) 
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:886) 
Caused by: kotlinx.serialization.MissingFieldException: Field 'onlyNotifyIfNoReviews' is required for type with serial name 'com.ichi2.anki.reviewreminders.ReviewReminder', but it was missing
	at kotlinx.serialization.internal.PluginExceptionsKt.throwMissingFieldException(PluginExceptions.kt:20)
	at com.ichi2.anki.reviewreminders.ReviewReminder.<init>(ReviewReminder.kt:190)
	at com.ichi2.anki.reviewreminders.ReviewReminder.<init>(Unknown Source:6)
	at com.ichi2.anki.reviewreminders.ReviewReminder$$serializer.deserialize(ReviewReminder.kt:190)
	at com.ichi2.anki.reviewreminders.ReviewReminder$$serializer.deserialize(ReviewReminder.kt:190)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
	... 28 more
```

--- a/AnkiDroid/src/main/java/com/ichi2/anki/reviewreminders/ReviewReminder.kt
+++ b/AnkiDroid/src/main/java/com/ichi2/anki/reviewreminders/ReviewReminder.kt
@@ -197,7 +197,7 @@ data class ReviewReminder private constructor(
     val scope: ReviewReminderScope,
     var enabled: Boolean,
     val profileID: String,
-    val onlyNotifyIfNoReviews: Boolean,
+    val onlyNotifyIfNoReviews: Boolean = false,
 ) : Parcelable,
     ReviewReminderSchema {
     companion object {

